python/smqtk/utils/postgres.py

```diff
@@ -87,18 +91,32 @@ def __init__(self, db_name='postgres', db_host=None, db_port=None,
         self.table_upsert_lock = table_upsert_lock
         self.table_upsert_sql = None
 
+        key_tuple = (
+            self.db_name,
+            self.db_user,
+            self.db_pass,
```

Comment:
The key can probably do without the password in it (the username should be sufficient).  I'm also not sure if this would be of security concern because a password may be held in memory.

Reply:
Passwords end up getting held in memory no matter what you do. That is unavoidable, and is taken care of by process isolation. That said, I agree that we can probably take it out of the key, since it doesn't really add anything unique to the identifier.